console/network/src/mainnet_v0.rs

```diff
@@ -137,26 +137,28 @@ impl Network for MainnetV0 {
     /// A list of (consensus_version, block_height) pairs indicating when each consensus version takes effect.
     /// Documentation for what is changed at each version can be found in `ConsensusVersion`.
     #[cfg(not(any(test, feature = "test")))]
-    const CONSENSUS_VERSION_HEIGHTS: [(ConsensusVersion, u32); 7] = [
+    const CONSENSUS_VERSION_HEIGHTS: [(ConsensusVersion, u32); 8] = [
         (ConsensusVersion::V1, 0),
         (ConsensusVersion::V2, 2_800_000),
         (ConsensusVersion::V3, 4_900_000),
         (ConsensusVersion::V4, 6_135_000),
         (ConsensusVersion::V5, 7_060_000),
         (ConsensusVersion::V6, 7_560_000),
         (ConsensusVersion::V7, 7_570_000),
+        (ConsensusVersion::V8, 9_850_000),
```

Comment:
Can we pull this forward by a week to 9,636,000 ? If we can land this by 7/29/2025, then we have the staking requirements in by end of July: https://github.com/ProvableHQ/snarkVM/pull/2734/files#diff-27286723a8ff091e27f327b158e5b9caf7d1e32e72307c769e712604fec9c524R27